podcasts/BookmarkManager.swift

```diff
@@ -0,0 +1,86 @@
+#if DEBUG
+    #if canImport(AudioToolbox)
+    import AudioToolbox
+    #endif
+#endif
+
+import Foundation
+import PocketCastsDataModel
+import PocketCastsUtils
+
+struct BookmarkManager {
+    typealias Bookmark = BookmarkDataManager.Bookmark
+
+    private let dataManager = DataManager.sharedManager.bookmarks
```

Comment:
I think it would be better to inject that through the constructor. This way if we want to unit test it, it's easy.